### ISSUE TITLE
Add comment with previous bytes/instruction in `wa`

### ISSUE THREAD

**Issue by radare** [radare/radare2#5768]

**Comment by radare** [comment]
inspiration source: https://twitter.com/keystone_engine/status/776116480236593152

**Comment by gtors** [comment]
How comment should be look?

```
[0x004049a0]> "wa nop;nop;"
Original 2 bytes (xor ebp, ebp)  <--------------------| Like this or what?
Written 2 bytes (nop; nop;) = wx 9090
``` 

Or comment should be placed into disassembled code? As here:
 ![comment_in_code](https://pbs.twimg.com/media/CsVOTZxVIAAumDX.jpg:large)

**Comment by Maijin** [comment]
I would add a "patch:" 
if short X bytes (xor ebp, ebp;XX) 
if long CCu base64 of X bytes (xor ebp, ebp;XX)

**Comment by radare** [comment]
This info can be genwrated at runtime from the write cache list that stores all write operations that happen in r2land. I would also base this with the bytes, not the disassembly; because code can be asembled in many different ways and when patching we want to be able to see the orog bytes to bring them back.

Comments will surive across multiple sessions if used with projects..

**Comment by radare** [comment]
ping ?  @gtors are you working on this?

**Comment by gtors** [comment]
@radare Not yet.

**Comment by gtors** [comment]
@radare 

> This info can be genwrated at runtime from the write cache list that stores all write operations that happen in r2land.

But what if `io.cache = false` ?

**Comment by radare** [comment]
then you have to open the file with -w or reopen with oo+ to get write permissions

**Comment by radare** [comment]
Fixed in ffcdb0443e1f92fabd2b60b2c914de866c7dc8fc
